luda-adventure/src/app/game/types/game_object/types/positioner/movement.rs

```diff
@@ -53,10 +55,20 @@ impl Movement {
             start_position: position,
             end_position: position,
             velocity: Xy::single(Per::new(0.tile(), 1.ms())),
+            movement_state: MovementState::Stuck,
```

Comment:
I think stay-forever is not stucking. And I believe you have wrong naming for `movement_state`.
`movement_state` should be `collision_state` because it is meaning about what kind of state this movement has for collision, to calculate [what you said](https://github.com/NamseEnt/namseent/pull/359#discussion_r988536420)

Reply:
Oh

Reply:
[f6fc994](https://github.com/NamseEnt/namseent/pull/359/commits/f6fc994268644b4ef995d238b4a2401e3b73506d)

Reply:
[1bb6880](https://github.com/NamseEnt/namseent/pull/359/commits/1bb68800a7d9dc124477986a6969d78875fa4d4b)